modules/sheet/tags/renderTags.js

```diff
@@ -0,0 +1,71 @@
+var { h, map, computed, Value, lookup: mutantLookup } = require('mutant')
+var nest = require('depnest')
+
+exports.needs = nest({
+  'about.obs.name': 'first',
+  'intl.sync.i18n': 'first',
+  'keys.sync.id': 'first'
+})
+
+exports.gives = nest('sheet.tags.renderTags')
+
+exports.create = function (api) {
+  const i18n = api.intl.sync.i18n
+  return nest('sheet.tags.renderTags', function (ids, select) {
+    var currentFilter = Value()
+    var tagLookup = mutantLookup(ids, (id) => {
+      return [id, api.about.obs.name(id)]
+    })
+    var filteredIds = computed([ids, tagLookup, currentFilter], (ids, lookup, filter) => {
```

Comment:
I notice this code has been repeated. You could extract it to scuttleTag?

Reply:
Which code specifically are you referring to here? Filtering the tags?

Reply:
I think he may be referring to `filteredIds`, repeated [here](https://github.com/ssbc/patchwork/pull/748/files#diff-e077b5d0ca4384194ede4fe34c2c19beR26).